port: "/dev/ttyUSBherku"
servos: 
    - 0 
    - 1 
    - 2 
    - 3
default_angles:
    - 0
    - 0
    - 0
    - 0
